### PR TITLE
Improve user feedback and speed-up interlis import qgep

### PR DESCRIPTION
**Pull request by sjib**
- [x] add messages in message bar
- [x] add messages to log
- [x] add Refresh materialized views
- [x] add main_cover function
- [x] add main_node function

Datamodel - see https://github.com/QGEP/datamodel/pull/235/commits/1c9079167b8f5b2e92bf2a8cfe393955acac24b0
- [x] add main_cover function
- [x] add main_node function

### CODE REVIEW

**Comment by sjib** [comment]
Should solve https://github.com/QGEP/qgepqwat2ili/issues/119

**Comment by sjib** [comment]
import_krattigen_dss_003_12052024_215355

It looks like the triggers are re-enable before the data is imported - the wizard is still open - see logging on screen shot.

@domi4484 Is this a similar error as you have found in TEKSI wastewater? https://github.com/teksi/wastewater/pull/211/files

How could the TEKSI improvment be backported to qgep?

**Comment by domi4484** [comment]
@sjib Yes it seems similar. Basically you need to re-enable the triggers only after the dialog has been acceptedd/rejected.

By re-thinking on this we could have an issue (in TEKSI as well) if someone kills QGIS before the trigger are re-enabled they will remain disabled forever.
Maybe we should add a check on startup of the plugin or some other mechanism to check if the trigger are enabled

**Comment by sjib** [comment]
- [x] Changed all post-processing to extra function `qgep_postimport` 

With that the triggers are enabled after the import - this speeds up the import greatly (e.g. Testdata K from @urskaufmann in 3 minutes instead of 30 minutes or more).

This needs these datamodel changes 1.6.2 to work properly!

File: qgepqwat2ili/postimport.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    logger.info("symbology triggers successfully created! (postimport.py)")
+    post_session.commit()
+    post_session.close()
+
+    #11.5.2024 add post_session2 - to do add queries for main_cover and main_node as in TEKSI, add to symbology functions
+    # see teksi ww https://github.com/teksi/wastewater/blob/3acfba249866d299f8a22e249d9f1e475fe7b88d/datamodel/app/symbology_functions.sql#L290
+    # needs delta_1.6.3_functions_update_fk_main_cover_main_wastewater_node.sql
+
+
+    post_session2 = Session(utils.sqlalchemy.create_engine(), autocommit=False, autoflush=False)
```
[the remark this answers is not in the document]
Comment (sjib) [reply qgepqwat2ili/postimport.py]: ok, see https://github.com/QGEP/qgepqwat2ili/pull/141/commits/165631ab1dc8bdcfd0472ba1a97481d50f2db132

File: qgepqwat2ili/gui/gui_import.py [hunk 2]
```diff
@@ -213,9 +216,24 @@ def commit_session(self):
...
         self.session.commit()
+
         self.session.close()
         iface.messageBar().pushMessage("Sucess", "Data successfully imported", level=Qgis.Success)
+        
+        # 31.5.2024 add post session - in postimport.py machen stattdessen
+        # init qgep_postimport
+        iface.messageBar().pushMessage("Info", "Start postimport", level=Qgis.Info)
+        
+        qgep_postimport()
```
[the remark this answers is not in the document]
Comment (sjib) [reply qgepqwat2ili/gui/gui_import.py:234]: yes.

[the remark this answers is not in the document]
Comment (sjib) [reply qgepqwat2ili/gui/gui_import.py:233]: https://github.com/QGEP/qgepqwat2ili/pull/141/commits/77851dd2b8c8f5f776e14545a7fb0dcb28735046

File: qgepqwat2ili/gui/gui_import.py [hunk 3]
```diff
@@ -213,9 +216,24 @@ def commit_session(self):
         # TODO : rollback to pre-commit state, allowing user to try to fix issues
         # probably a matter of creating a savepoint before saving with
         # session.begin_nested() and one additionnal self.session.commit()
+        
+        # 11.5.2024 try to add info in message bar
+        iface.messageBar().pushMessage("Please be patient!", "Importing data in qgep - working ...", level=Qgis.Warning)
+        time.sleep(5)
```
[the remark this answers is not in the document]
Comment (sjib) [reply qgepqwat2ili/gui/gui_import.py]: Just testing - took it out with https://github.com/QGEP/qgepqwat2ili/pull/141/commits/77851dd2b8c8f5f776e14545a7fb0dcb28735046

**Comment by sjib** [comment]
@ponceta Ready for review and then merging - needs these https://github.com/QGEP/datamodel/pull/235 to work properly!

**Review by ponceta — APPROVED** [review]

**Pull request merged**